packages/demo-react/src/App.js

```diff
@@ -22,7 +22,7 @@ function App() {
       </p>
       <div className="App-spacer" />
       <div className="App-container">
-        <PopupButton id="moe6aa" style={{ padding: 12, fontSize: 16 }} size={80}>
+        <PopupButton asHtml={null} id="moe6aa" style={{ padding: 12, fontSize: 16 }} size={80}>
```

Comment:
What is `asHtml` prop? If you want to make an example if `as` prop please add it as a separate example below this one. Keep this one without the `as` prop.

Reply:
Updated, thanks!